resources/qml/Actions.qml

```diff
@@ -188,6 +189,13 @@ Item
         shortcut: "Ctrl+K"
     }
 
+    Action
+    {
+        id: marketplaceMaterialsAction
+        iconName: "configure"
```

Comment:
Why this iconName? Does it doe anything? Don't think it is mandatory, when looking at other usages

Reply:
These icons are visible on some operating systems. It displays a little icon then at the context menu option.